stripe/_subscription.py

```diff
@@ -2363,7 +2363,7 @@ class SearchParams(RequestOptions):
     """
     A date in the future at which the subscription will automatically get canceled
     """
-    cancel_at_period_end: Optional[bool]
```

Comment:
also this one

Reply:
This should have gone in as part of prior PRs - not sure why it got left out - yes this is intentional - it is one of the overrides we added